Dockerfile.build.tpl

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6-alpine
```

Comment:
Official Python image is based on alpine 3.4, but we're using alpine 3.5 instead for every image. So use `alpine:3.5` as base image.